### PR TITLE
feat: remove optional tag

### PR DESCRIPTION
**Pull request by mpodlasin**
Relates to https://github.com/stoplightio/elements/issues/655

### CODE REVIEW

File: src/components/shared/Validations.tsx [hunk 1]
```diff
@@ -18,9 +17,9 @@ export const Validations: React.FunctionComponent<IValidations> = ({
   const validationCount = Object.keys(validations).length;
 
   const requiredElem = (
-    <div className={cn('ml-2', required ? 'font-medium' : 'text-darken-7 dark:text-lighten-6')}>
-      {required ? 'required' : 'optional'}
-      {validationCount ? `+${validationCount}` : ''}
+    <div>
+      {required && <span className="ml-2 text-orange-7 dark:text-lighten-6">required</span>}
```
Comment (P0lip) [inline src/components/shared/Validations.tsx:21]: image
```suggestion
      {required && <span className="ml-2 text-orange-7 dark:text-orange-6">required</span>}
```

**Review by P0lip — CHANGES_REQUESTED** [review]
Let's tweak the color a bit.

File: src/components/shared/Validations.tsx (hunk 1 above)
Comment (mpodlasin) [reply src/components/shared/Validations.tsx]: uups, didn't notice that.

Fixed, thanks for catching it :)

**Comment by mmiask** [comment]
I know it's probably out of scope for this PR, but we might want to come up with a better solution for displaying validations when they are not directly rendered on the right side of the row

e.g.

Screenshot 2020-11-04 at 13 30 43
 
For `email`,  this `+3` looks a little weird to me, when there are no validations displayed directly on the row. I would suggest rendering either name of a first validation + number of other validations e.g. `min-length + 4` or `validations: count`

**Comment by philsturgeon** [comment]
Right on @mmiask, we've planned a refresh for this area which is being handled by this issue: https://github.com/stoplightio/elements/issues/668

**Review by P0lip — APPROVED** [review]

**Pull request merged**